File: js/al18.js
const m1 = [
    6,
    0,
];
const m2 = [
    7,
    2,
];


let sa1 = m1[1]-m2[1];
let sa2 = m1[0]-m2[0];

let a = sa1/sa2;

let b = m1[1] + a * m2[0];

console.log( "y = "+ a + "x + "+ b);

File: python /al18.py
m1 = {
    "x1": 6,
    "y1": 0,
}

m2 = {
    "x2": 7,
    "y2": 2,
}

sa1 = m1["y1"] - m2["y2"]
sa2 = m1["x1"] - m2["x2"]
a = sa1 / sa2

b = m1["y1"] + a * m1["x1"]

print("y = " + str(a) + "x + " + str(b))